README.md

```diff
@@ -67,32 +56,34 @@ az keyvault secret show --name CosmosDatabase --vault-name $He_Name
 
 ```
 
-### Run the app
+### Using Visual Studio Codespaces
 
-> A debug build can connect to Key Vault using MSI, Azure CLI or Visual Studio Credentials
+- Open `launch.json` in the `.vscode` directory
+- Replace `{your key vault name}` with the name of your key vault
+  - the file saves automatically
+- Press F5
+- Wait for `Application started. Press Ctrl+C to shut down.` in the Debug Console
+- Skip to the testing step below
 
-- If you are using CodeSpaces, you just need to set the --keyvault-name parameter and press F5
-  - Open launch.json in the .vscode directory at the root of the repo
-  - Replace `{your keyvault name}`
+### Using bash shell
 
+> This will work from a terminal in Visual Studio Codespaces as well
 
 ```bash
 
-# make sure you are in the root of the repo
-cd src/app
-dotnet restore
+# run the application
+# He_Name was set during setup and is your Key Vault name
+dotnet run -p src/app/helium.csproj -- --auth-type CLI --keyvault-name $He_Name
 
-# run in the background
-# $He_Name is set to the name of your key vault during setup
-# this will use Azure CLI cached credentials
+```
 
-dotnet run -- --keyvault-name $He_Name --auth-type CLI
+wait for `Application started. Press Ctrl+C to shut down.`
 
-# press ctl-c to stop the app
+### Testing the application
 
-```
+Open a new bash shell
 
-### In a separate terminal
+> Visual Studio Codespaces allows you to open multiple shells by clicking on the `Split Terminal` icon
 
```

Comment:
Were we going to include a reference to httpie? Now I forget how what we decided at standup...

Reply:
good catch - updated